lib_news/models.py

```diff
@@ -1,6 +1,9 @@
 import json
 from urllib.request import URLError, urlopen
 
+from django.contrib.syndication.views import Feed
+from django.http import HttpResponse
```

Comment:
I'm seeing linting errors in this file. These are mostly unused imports and whitespace errors.